bin/lib/lock_webtask.js

```diff
@@ -0,0 +1,155 @@
+'use latest';
+
+const hbs         = require('handlebars');
+const url         = require('url');
+const qs          = require('qs');
+const request     = require('request');
+const express     = require('express');
+const fromExpress = require('webtask-tools').fromExpress;
+
+const app = express();
+
+/** Webtask proxy to authenticate with auth0
+ * @param container task tenant
+ * @param taskname task to authenticate
+ * @param account Auth0 account to use
+ */
+
+const VIEW = hbs.compile(`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="utf8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
+        <link rel="shortcut icon" href="https://webtask.io/images/favicon/favicon.ico">
+
+        <title>Authenticate</title>
+
+        <script src="//cdn.auth0.com/js/lock-7.6.2.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/then-request/2.1.1/request.min.js"></script>
+      </head>
+
+      <body>
+        <script type="application/javascript">
+            window.TASK_URL       = '{{{TASK_URL}}}';
+            window.CLIENT_ID      = '{{{CLIENT_ID}}}';
+            window.AUTH0_DOMAIN   = '{{{AUTH0_DOMAIN}}}';
+            window.REQ_METHOD     = '{{{REQ_METHOD}}}';
+
+            window.onload = {{{FUNC_TO_RUN}}};
+        </script>
+      </body>
+    </html>
+`);
+
+const FUNC_TO_RUN = function() {
+    var lock = new Auth0Lock(CLIENT_ID, AUTH0_DOMAIN);
+    var lock_opts = {
+        authParams: {
+            scope: 'openid email'
+        }
+    };
+
+    lock.show(lock_opts, function (err, profile, id_token) {
+        if(err) return console.log(err);
+
+        var opts = {
+            headers: {
+                'Authorization': 'Bearer ' + id_token
+            }
+        };
+
+        request(REQ_METHOD, TASK_URL, opts)
+            .then(function (res) {
+                switch(res.headers['content-type']) {
+                    case 'application/json':
+                        var parsed = JSON.parse(res.body);
+
+                        document.body.innerHTML = '<pre>' + JSON.stringify(parsed, null, 1) + '</pre>';
+                        break;
+                    default:
+                        document.body.innerHTML = res.body;
+                }
+            })
+            .catch(function (err) {
+                console.log(err);
+            });
+    });
+}.toString();
+
+function getAuthToken (req) {
+    if(req.query.key)
+        return req.query.key;
+
+    if(req.headers.authorization)
+        return req.headers.authorization
+            .match(/^bearer\s+(.+)$/i)[1];
+
+    return false;
+}
+
+const query_delete = [
+    'baseUrl',
+    'container',
+    'taskname',
+    'domain',
+    'clientId',
+];
+
+app.get('*', (req, res) => {
```

Comment:
Is there any value in pulling in express machinery if you have just one handler?